Dockerfile.dev

```diff
@@ -1,21 +1,13 @@
-FROM mcr.microsoft.com/vscode/devcontainers/python:0-3.9
+FROM mcr.microsoft.com/devcontainers/python:0-3.9
 
-SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+# Ensure RUN fails if any command fails
+SHELL ["/bin/bash", "-euxo", "pipefail", "-c"]
 
-# Uninstall pre-installed formatting and linting tools
-# They would conflict with our pinned versions
-RUN pipx uninstall black
-RUN pipx uninstall flake8
-RUN pipx uninstall pydocstyle
-RUN pipx uninstall pycodestyle
-RUN pipx uninstall mypy
-RUN pipx uninstall pylint
-
-RUN \
-    curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - \
-    && apt-get update \
-    && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
-        # Additional library needed by some tests and accordingly by VScode Tests Discovery
+# Install apt-get dependencies
+RUN export DEBIAN_FRONTEND=noninteractive; \
```

Comment:
Please chain Docker commands using `&&`

I know you change the shell properties above, however, this is not how general standards are in Dockerfile. Please revert that non-common code style.

Reply:
Ok.